modules/monitoring/application/views/scripts/partials/downtime/downtime-header.phtml

```diff
@@ -10,21 +10,41 @@
 <td>
   <div class="comment-author">
     <?php if ($isService): ?>
-        <?= $this->icon('service', $this->translate('Service')); ?> <?= $this->qlink(
-            $downtime->host_display_name . ': ' . $downtime->service_display_name,
-            'monitoring/service/show',
-            array(
-                'host'    => $downtime->host_name,
-                'service' => $downtime->service_description
+    <?php
+        echo $this->qlink(
```

Comment:
If you really don't want to use `<?= ... ?>` (I fail to see why) please pay attention to indentation.